arcaflow_plugin_pcp/pcp_plugin.py

```diff
@@ -127,12 +138,14 @@ def start_pcp(
         pcp2_flags = [
             "-a",
             "pmlogger-out",
+            "-t",
+            str(params.pmlogger_interval),
             "-f",
             # pmrep doesn't accept %z for the timezone,
             # so we'll get it explicitly via datetime
-            f"%FT%T.%f{datetime.now().astimezone():%z}",
+            f"%FT%T.%f{str(datetime.now().astimezone())[-6:]}",
```

Comment:
Why did this regress to the substring extraction instead of `{...:%z}`??

Reply:
It turns out that Horreum expects the time format to match the RFC 3339 spec, which needs the colon in the time zone offset. ISO 8601 will accept either the `+0000` or the `+00:00` format, but RFC 3339 is more strict.

Reply:
Dang. Python 3.12 adds `%:z` format which follows RFC3339 (`±HH:MM[:SS[.ffffff]]`), but I believe Arcaflow is still using 3.10.

Reply:
> I believe Arcaflow is still using 3.10.

Or `3.9`, even....

Reply:
I searched for a better way to do this than string manipulation, but I came up dry (and frustrated). In a python console, it possible to pass a `:%:z` format parameter successfully to the f-string, which is apparently a GNU semantic for inserting the colon, but same approach did not work with the containerized plugin.

Reply:
> Or `3.9`, even....

Pbench uses 3.9, but I thought ARCA was on 3.10? In any case, it doesn't matter.

> In a python console, it is possible to pass a :%:z format parameter

So you're running 3.12 locally even though your container isn't; which is fine until it isn't. 😆